src/modules/renderTodoList.js

```diff
@@ -0,0 +1,96 @@
+import updateLocalStorage from './updateLocalStorage.js';
+import { handleDragEnter, handleDragOver, handleDragStart, handleDrop, handleDragEnd, handleDragLeave } from './dragAndDrop.js';
+
+export const tasks = [];
+  
+const renderTodoList = () => {
+  const todoListElement = document.getElementById('todo-list');
+  todoListElement.innerHTML = ''
+
+  tasks.forEach((task, index) => {
+    const listItem = document.createElement('li');
+    listItem.setAttribute('data-index', index);
+
+    listItem.setAttribute('draggable', true);
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.classList.add('checkedBtn')
+    checkbox.checked = task.completed;
+    checkbox.addEventListener('change', () => {
+      task.completed = checkbox.checked;
+      updateLocalStorage();
+      renderTodoList();
+    });
+
+    const input = document.createElement('input');
+    input.classList.add('items');
+    input.type = 'text';
+    input.value = task.description;
+    input.addEventListener('keydown', (event) => {
+      if (event.key === 'Enter') {
+        const newDescription = input.value.trim();
+        if (newDescription !== '') {
+          task.description = newDescription;
+          updateLocalStorage();
+          renderTodoList();
+        }
+      }
+    });
+
+    
+
+    const deleteButton = document.createElement('button');
+    deleteButton.classList.add('delete-btn');
+    deleteButton.innerHTML = '<i class="ri-delete-bin-line"></i>';
+
+    deleteButton.addEventListener('click', () => {
+      tasks.splice(index, 1);
```

Comment:
Great job on deleting a task :star_struck: . If you use **array.filter()** method here it will be much more efficient. Don't you think?